pkgs/os-specific/linux/open-iscsi/default.nix

```diff
@@ -1,22 +1,23 @@
-{ stdenv, fetchurl, nukeReferences }:
+{ stdenv, fetchgit, nukeReferences, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod}:
 let
   pname = "open-iscsi-2.0-873";
```

Comment:
The important thing about version strings is that `old < new` holds for all `old` and `new`, so I'm not such a huge fan of appending revisions, but it is done elsewhere. If you do want to use the revision, please use something like `lib.substring 0 7 rev`